Source/WebCore/rendering/RenderBlockFlow.cpp

```diff
@@ -91,6 +91,37 @@ struct SameSizeAsMarginInfo {
 static_assert(sizeof(MarginValues) == sizeof(LayoutUnit[4]), "MarginValues should stay small");
 static_assert(sizeof(RenderBlockFlow::MarginInfo) == sizeof(SameSizeAsMarginInfo), "MarginInfo should stay small");
 
+namespace BlockStepSizing {
```

Comment:
I would probably move this to a `BlockStepSizing.h` file.

Reply:
I actually somewhat prefer having it live in `RenderBlockFlow` since that's what will be using of it as right now, but I will definitely keep this idea in mind!